lib/gnat/jetstream/pager.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Gnat.Jetstream.Pager do
+  @moduledoc false
+
+  alias Gnat.Jetstream
+  alias Gnat.Jetstream.API.{Consumer, Util}
+
+  @opaque pager :: map()
+  @type message :: Gnat.message()
+
+  def init(conn, stream_name, opts) do
+    name = "gnat_stream_pager_#{Util.nuid()}"
+
+    first_seq = Keyword.fetch!(opts, :from_seq)
```

Comment:
I imagine that we would end up allowing other options like `since: utc_datetime` or `limit: 1000` in these opts